.github/workflows/test.yml

```diff
@@ -12,48 +12,56 @@ jobs:
       matrix:
         os: [ ubuntu-latest, macos-latest ]
         python-version: [ 3.8 ]
-
+    defaults:
+      run:
+        shell: bash -l {0}
     steps:
       - uses: actions/checkout@v2
 
       - uses: conda-incubator/setup-miniconda@v2
         with:
           auto-update-conda: true
           python-version: ${{ matrix.python-version }}
-          auto-activate-base: true
           channels: conda-forge
 
       - name: Install deps
-        shell: bash -l {0}
         run: |
-          conda install mamba pip
-          conda activate base
+          conda activate test
```

Comment:
@mariusvniekerk we do this by default for you: so you can have 3 cases:

Either you use `base` by default, in which case you need

## Activate `base` for all steps

```yaml
 - uses: conda-incubator/setup-miniconda@v2
        with:
          auto-update-conda: true
          python-version: ${{ matrix.python-version }}
          auto-activate-base: true	
          activate-environment: ""
          channels: conda-forge
```

## Activate `whatever` for all steps

```yaml
 - uses: conda-incubator/setup-miniconda@v2
        with:
          auto-update-conda: true
          python-version: ${{ matrix.python-version }}
          auto-activate-base: true
          activate-environment: whatever
          channels: conda-forge
```


## Activate `test` for all steps

```yaml
 - uses: conda-incubator/setup-miniconda@v2
        with:
          auto-update-conda: true
          python-version: ${{ matrix.python-version }}
          auto-activate-base: true
          channels: conda-forge
```